.vsts/builds/ci-official.yml

```diff
@@ -18,11 +18,19 @@ phases:
   variables:
     CI: true
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+    TeamName: AspNetCore
+    _SignType: real
```

Comment:
is this just to signify its a private var used in this yml only?  just curious what the pattern is

Reply:
Also what are the other sign types, test? (maybe worth a comment for posterity)

Reply:
SignType = { test, real }

I actually used the underscore for a different reason. VSTS variables automatically become environment variables, and this one was causing issues in MSBuild projects which use the `$(SignType)` msbuild prop.

Reply:
Ok cool, good to know :)

Reply:
I added this to the yml as comments for future reference.